compio-io/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "compio-io"
+version = "0.1.0"
+categories = ["asynchronous"]
+keywords = ["async", "io"]
+edition = { workspace = true }
+authors = { workspace = true }
+readme = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+
+[dependencies]
+compio-buf = { workspace = true, features = ["arrayvec"] }
+paste = "1.0.14"
+
+[dev-dependencies]
+compio = { path = "../compio", features = ["all"] }
```

Comment:
Why?

Reply:
I was writing tests

Reply:
Won't it cause cyclic reference?

If you need many components, it is better to write the test in compio crate.

Reply:
Sure